lizard_connector/connector.py

```diff
@@ -124,27 +143,9 @@ def perform_request(self, url, data=None):
             request_obj = urllib_request.Request(url, headers=self.header)
```

Comment:
#141: method="POST"
Not backwards compatible with Python 2.
```
======================================================================
ERROR: test_post (test.test_connector.ConnectorTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/madeleine.vanwinkel/code/lizard-connector/test/test_connector.py", line 75, in test_post
    self.__connector_test(self.connector.post, 'https://test.nl', {'data': 1})
  File "/home/madeleine.vanwinkel/code/lizard-connector/test/test_connector.py", line 67, in __connector_test
    return connector_method(*args, **kwargs)
  File "/home/madeleine.vanwinkel/code/lizard-connector/lizard_connector/connector.py", line 117, in post
    return self.perform_request(url, data)
  File "/home/madeleine.vanwinkel/code/lizard-connector/lizard_connector/connector.py", line 141, in perform_request
    method="POST")
TypeError: __init__() got an unexpected keyword argument 'method'
```

Reply:
Dit is gefixt.